src/lib/wasm/TesseractStore.ts

```diff
@@ -39,7 +39,7 @@ class TesseractStore {
         try {
             if (tesseract) {
                 tesseract.unlock(passphrase)
```

Comment:
This should yield an error if it fails

Reply:
Tesseract#unlock seems to return void. at least for wasm

Reply:
Correct. On Rust, it returns `Result<(), Error>` but on js/rs it would return `void` but internally may throw an error when the rust code returns an error.